src/s3ql/backends/b2/b2_backend.py

```diff
@@ -515,8 +515,7 @@ def _write_fh(self, key: str, fh: BinaryIO, off: int, len_: int, metadata: Dict[
             self.upload_connection = None
             raise
 
-        json_response = json.loads(response_body.decode('utf-8'))
-        return json_response
+        return len_
```

Comment:
I believe this should return the size of the object that was created on the server-side. What you are returning is the size of the "success, I stored this object" message.

Reply:
I don't think len_ is the "success" message.  It is the original parameter of the function that is the content length of the blob.
ie: `headers['Content-Length'] = len_`
and I checked the JSON blob reply text, and its length field matched the len_ parameter.

The length of the response would be something like len(response_body) ?